### PR TITLE
Updating bootstrap - Deimos

### PR DESCRIPTION
**Pull request by Deimos620**
from Bootstrap 2.0.3 to 3.2.2

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -57,3 +57,62 @@
   end
 
 end
+# This file should contain all the record creation needed to seed the database with its default values.
```
Comment (spacemunkay) [inline db/seeds.rb]: It looks like some code got copy pasted twice?

File: README.md [hunk 2]
```diff
@@ -1,50 +1,41 @@
-# About
```
Comment (spacemunkay) [inline README.md]: Can you undo your changes to the README?  You removed the link to the software license, docker instructions, and other important info.  

Instead, I think you just need to add your Step 5 instruction about the bootstrap commands to the following installation guide file: https://github.com/spacemunkay/BikeShed/blob/master/doc/local_dev_setup.md

**Comment by spacemunkay** [comment]
I noticed issues with the following two pages being non-responsive:

/time_entries/new
screen shot 2016-07-08 at 3 53 07 pm

/bikes/new
screen shot 2016-07-08 at 3 53 23 pm

Do they render the same way for you?  Could you update these templates to correctly use Bootstrap 3?

In addition, I can no longer click buttons or submit any of the forms.  Do you also have this problem?

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -1,35 +1,42 @@
...
+      .container
+        .navbar-header
+          %button.navbar-toggle(type="button" data-toggle="collapse" data-target=".navbar-responsive-collapse")
+            %span.icon-bar
+            %span.icon-bar
+            %span.icon-bar
+          %a.navbar-brand(href="#") Samaritan
+        .navbar-collapse.collapse.navbar-responsive-collapse
+          %ul.nav.navbar-nav
+            %li= link_to "Link 1", "/path1"
```
Comment (spacemunkay) [inline app/views/layouts/application.html.haml]: I'm curious where this code snippet came from?

File: app/views/layouts/application.html.haml [hunk 4]
```diff
@@ -1,35 +1,42 @@
...
+            %li= link_to "Link 2", "/path2"
+            %li= link_to "Link 3", "/path3"
+            %li= link_to "Link 4", "/path4"
+    .container.panel-body
+      .row
+        .col-lg-6.col-lg-offset-3
+          = bootstrap_flash
+          = yield
+          %footer
+            %p &copy; Company 2016
```
Comment (spacemunkay) [inline app/views/layouts/application.html.haml]: Lost BikeShed here

File: app/views/layouts/application.html.haml [hunk 5]
```diff
@@ -1,35 +1,42 @@
...
-    = javascript_include_tag params[:controller] 
+  %body
+    .navbar.navbar-default.navbar-fixed-top
+      .container
+        .navbar-header
+          %button.navbar-toggle(type="button" data-toggle="collapse" data-target=".navbar-responsive-collapse")
+            %span.icon-bar
+            %span.icon-bar
+            %span.icon-bar
+          %a.navbar-brand(href="#") Samaritan
```
Comment (spacemunkay) [inline app/views/layouts/application.html.haml]: Where did Samaritan come from?

File: .idea/.generators [hunk 6]
```diff
@@ -5,4 +5,4 @@ You are allowed to:
 2. Remove generators
```
Comment (spacemunkay) [inline .idea/.generators]: Are these `.idea/*` files auto-generated?  Could you remove them and add the following to the `.gitignore` file?  
```
.idea
.idea/*
```

File: .idea/.generators [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (spacemunkay) [inline .idea/.generators]: Thanks for adding the files to the `.gitignore`.  Could you remove the rest of the .idea files?

File: app/views/time_entries/index.haml [hunk 8]
```diff
@@ -22,23 +23,24 @@
...
-  .modal-footer
-    %button{ class: "btn", "data-dismiss" => "modal", "aria-hidden" => "true"} Close
-    %button{ id: "confirmation_delete", class: "btn btn-primary", "data-url" => "api/v1/time_entries/" } Delete
+%a{class: "btn btn-block btn-primary", href: new_time_entry_path } Add Time Entry
+%br
+.modal{ id: "confirmation", class: "modal-fade", role: "dialog", "aria-labelledby" => "confirmation_title", "aria-hidden" => "true", tabindex: -1 }
+  .modal-dialog
+    .modal-content
+      .modal-header
+        %button{ class: "close", "data-dismiss" => "modal", "aria-hidden" => "true"}x
```
Comment (spacemunkay) [inline app/views/time_entries/index.haml]: For some reason the `x` is no longer visible in the red buttons:
screen shot 2016-07-13 at 7 26 16 am

Comment (Deimos620) [inline app/views/time_entries/index.haml]: Please let me know if you have any other problem. it will be fixed soon

**Comment by spacemunkay** [comment]
The `bikes/new` page still has the same issues as described earlier, can't click radio buttons and can't submit the form:
screen shot 2016-07-13 at 7 36 59 am.  I think it needs changes similar to the the time entry form you just fixed.

**Comment by spacemunkay** [comment]
Thanks for adding the .idea files to the .gitignore.  Could you remove the .idea files that were committed to the repository?  It's okay for you to use them locally on your computer, but I do not want them as part of the source code.

**Comment by spacemunkay** [comment]
Re-opened on https://github.com/spacemunkay/BikeShed/pull/112

**Pull request closed without merging**